RctGcs/rctGCS.py

```diff
@@ -106,6 +105,7 @@ def configSetup() -> Path:
 def main():
     start_timestamp = dt.datetime.now()
     log_file = start_timestamp.strftime('%Y.%m.%d.%H.%M.%S_gcs.log')
+    log_file = '0gcs.log'
```

Comment:
timestamp and filename based on it are unnecessary

Reply:
Resolved in 073c7e70b161e8ed94be64dd9ec874f96ad41076